src/js/confirmation/Confirmation.jsx

```diff
@@ -1,61 +1,50 @@
-import React, {PureComponent} from 'react';
+import React, {useEffect} from 'react';
 import PropTypes from 'prop-types';
 import {connect} from 'react-redux';
 import {push} from 'connected-react-router';
 import Button from '../common/Button';
 import Image from '../common/Image';
 
-class Confirmation extends PureComponent {
-    static propTypes = {
-        email: PropTypes.string.isRequired,
-        selectedSpot: PropTypes.object,
-        pushTo: PropTypes.func.isRequired,
-    };
-
-    constructor(props) {
-        super(props);
-
-        const {selectedSpot, pushTo} = props;
-
+const Confirmation = ({selectedSpot, pushTo, email}) => {
```

Comment:
Would it be better to dispatch pushTo here rather than passing down via props?  that said - keeping all changes in one file is probably worth it for this PR

Reply:
I think I'd prefer to keep the changes as minimal as possible